satpy/etc/readers/avhrr_l1b_gaclac.yaml

```diff
@@ -15,6 +15,9 @@ datasets:
             units: '%'
           radiance:
             standard_name: toa_outgoing_radiance_per_unit_wavelength
+          counts:
+            standard_name: counts
```

Comment:
Although it is used in the same way by other readers, `counts` is actually not a valid CF standard name

Reply:
Good Point and yes, I basically copied it from here /seviri_l1b_native.yaml. But this means all .yaml reader files with  `counts` would be need to be changed, no?

Reply:
Probably yes. But it's not a big deal I think. I'll file a satpy issue for that.